HW03/HW3.Rmd

```diff
@@ -0,0 +1,415 @@
+---
+title: "HW3"
+author: "han"
+date: "2/19/2021"
+output:
+    html_document: default
+    github_document: default
+    theme: paper
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(dplyr)
+library(tidyr)
+library(tibble)
+library(ggplot2)
+library(tidyverse)
+library(gridExtra)
+library(reshape2)
+library(ggdist)
+theme_set(theme_classic())
+dictionary <- readr::read_csv("green_dictionary.csv")
+green_data <- readr::read_csv("green_data.csv")
+```
+
+# Homework 03
+
+#### You will analyze data looking at the relationship between **green reputation** and three personality traits--**compassion**, **intellectual curiosity**, and **openness to experiences**.
+The dataset includes data from **students** and **non-students**.
+
+
+## For your assignment, do the following.
+
+### Q1: Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+
+     
+```{r Q1}
+
+# Is anything unusual?
+
+Q1set <- filter(green_data, id == 5549) #Searches id for value "5549" 
+Q1set.1 <- as.data.frame(t(as.matrix(Q1set) #Flips columns and rows
+                           )
```

Comment:
Awesome work! 
Your coding is very neat and easy to read. I like how you used the melt function and how you searched for the value "5549". I do think that it would look more clean and organized if you condensed your code a little more such as keeping the brackets together instead of spreading them out.